dev/com.ibm.ws.microprofile.config.1.1.cdi/src/com/ibm/ws/microprofile/config/cdi/ConfigCDIExtension.java

```diff
@@ -60,7 +62,13 @@ protected void addBadInjectionType(Type type) {
 
     void processInjectionTarget(@Observes ProcessInjectionTarget<?> pit) {
         Class<?> targetClass = pit.getAnnotatedType().getJavaClass();
-        ClassLoader classLoader = targetClass.getClassLoader();
+
+        ClassLoader classLoader = AccessController.doPrivileged(new PrivilegedAction<ClassLoader>() {
+            @Override
+            public ClassLoader run() {
+                return targetClass.getClassLoader();
```

Comment:
I think I saw this same call to get classloader in other versions of MP Config (besides just 1.1).  It might be good to update them all - when I looked, I saw it in 1.4, but it's a good guess that it needs to be updated in 1.2 and 1.3 too.

Reply:
I think Andy is right, there is another getClassLoader in `Config14CDIExtension`. The 1.2 and 1.3 versions don't have much in since they just inherit from this class.

Reply:
I've updated ```Config14CDIExtension``` as well, thanks for pointing it out.